pkg/service/catalog/api/types.go

```diff
@@ -0,0 +1,21 @@
+package api
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+type CatalogService struct {
+	// Target is a reference to a service
```

Comment:
These are virtual objects. You can't create them by hand. Do you think we need TypeMeta or ObjectMeta?